draft-ietf-quic-recovery.md

```diff
@@ -436,12 +436,13 @@ The RECOMMENDED initial value for the packet reordering threshold
 ({{?RFC5681}}, {{?RFC6675}}).  In order to remain similar to TCP,
 implementations SHOULD NOT use a packet threshold less than 3; see {{?RFC5681}}.
 
-Some networks may exhibit higher degrees of reordering, causing a sender to
-detect spurious losses.  Algorithms that increase the reordering threshold after
-spuriously detecting losses, such as TCP-NCR ({{?RFC4653}}), have proven to be
-useful in TCP and are expected to be at least as useful in QUIC.  Re-ordering
-could be more common with QUIC than TCP, because network elements cannot observe
-and fix the order of out-of-order packets.
+Some networks may exhibit higher degrees of packet reordering, causing a sender
+to detect spurious losses. Additionally, packet reordering could be more common
+with QUIC than TCP, because network elements that could observe and fix the
+order of reordered TCP packets cannot do it with QUIC. Algorithms that increase
```

Comment:
```suggestion
order of reordered TCP packets cannot do that for QUIC. Algorithms that increase
```

Reply:
Missed this. Fixed this in https://github.com/quicwg/base-drafts/commit/8abb9fa2354ae341637ce43479a707844aa4a7cf